pkg/gameservers/gameservers.go

```diff
@@ -24,6 +24,11 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+const (
+	// NodePodIP identifies an IP address from a pod.
+	NodePodIP corev1.NodeAddressType = "PodIP"
```

Comment:
I think it's reasonable to put the const here: https://github.com/googleforgames/agones/blob/4bb673186b2ee8431de00c5c564bd1daa1a356df/pkg/apis/agones/v1/gameserver.go#L114-L150 .